packages/cli-plugin-metro/src/commands/start/runPackager.ts

```diff
@@ -0,0 +1,33 @@
+import {isPackagerRunning, logger} from '@react-native-community/cli-tools';
+import {startServerInNewWindow} from './startServerInNewWindow';
+
+export async function runPackager(
+  port: number,
+  root: string,
+  reactNativePath: string,
+  terminal?: string,
+  packager?: boolean,
+) {
+  if (!packager) {
+    return;
+  }
+  const result = await isPackagerRunning(port);
+  if (result === 'running') {
+    logger.info('JS server already running.');
+  } else if (result === 'unrecognized') {
+    logger.warn('JS server not recognized, continuing with build...');
+  } else {
+    // result == 'not_running'
```

Comment:
Should we explicitly compare against `'not_running'`? What other results are possible?

Reply:
We should also log unhandled results in case there's something that needs to be explicitly handled later.